.codeclimate.yml

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
Codecov was raising a lot of noise, since we're no concerned with test coverage. Let's use codeclimate for coverage also.